File: BE/User/HanjanUserBE/src/main/java/com/hanjan/user/data/vo/UserVo.java
package com.hanjan.user.data.vo;

import com.hanjan.user.data.dto.KakaoAccountDto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@AllArgsConstructor
@Builder
public class UserVo {
	private long uid;
	private String kakao_id;
	private String name;
	private String nickname;
	private String image;
	private String birth;
	private String gender; 
	private String interest;
	private int reportcnt;
	
	
	public UserVo(KakaoAccountDto accountDto) {
		this.kakao_id = accountDto.getKakao_id();
		this.gender = accountDto.getGender();
		this.birth = accountDto.getBirth();
		this.image = accountDto.getImage();
	}
}

File: Android/app/src/main/java/com/hanjan/hanjangame/dto/UserVo.kt
package com.hanjan.hanjangame.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class UserVo(
    @JsonProperty("birth")
    val birth: String,
    @JsonProperty("gender")
    val gender: String,
    @JsonProperty("image")
    val image: String,
    @JsonProperty("kakao_id")
    val kakao_id: String,
    @JsonProperty("name")
    val name: String,
    @JsonProperty("interest")
    val interest: String,
    @JsonProperty("nickname")
    val nickname: String,
    @JsonProperty("reportcnt")
    val reportcnt: Int,
    @JsonProperty("uid")
    val uid: Int
)